v0/destinations/marketo/transform.js

```diff
@@ -197,7 +203,9 @@ const lookupLeadUsingId = async (
     const data = marketoResponseHandler(
       clientResponse,
       "[Marketo Transformer]: During lead look up using userId",
-      TRANSFORMER_METRIC.TRANSFORMER_STAGE.TRANSFORM
+      TRANSFORMER_METRIC.TRANSFORMER_STAGE.TRANSFORM,
+      authCache,
+      formattedDestination.ID
```

Comment:
Rather than scattering the auth cache key in multiple places in the code, we can abstract it as a function. So, there is only one place we need to change if we ever wish to change the key value.

Reply:
Can these parameters be abstracted as the `destinationId` is not accessible globally here?

Reply:
Yeah, I'd send the whole input event to that new function and let it figure out and return the auth cache key.

Reply:
Updated it.